bitrise.yml

```diff
@@ -1448,7 +1449,7 @@ workflows:
                 -scheme "$SCHEME" \
                 -sdk "$SDK" \
                 -destination "$DESTINATION" \
-                -derivedDataPath "${DERIVED_DATA_DIR}"
+                -derivedDataPath "${DERIVED_DATA_DIR}" > xcodebuild.log # do not display lengthy output
```

Comment:
I will remove `> xcodebuild.log` when this PR is ready to merge. Bitrise truncates the logs so that I could not see the errors after compilation.